projects/collections/schema.js

```diff
@@ -1,3 +1,76 @@
+//helper schemas for the projectschema dataStore field
+KeyStakeHolderSchema = new SimpleSchema({ //temporary schema of a certain category of users that will be implemented in later iterations
+  name: {
+    type:String,
+    label: "The name of the stakeholder"
+  },
+  "mediaId": { //used to store logos/images etc of the stakeholders
+    type: String,
```

Comment:
Should this be `[String]`, meaning 'array of strings', to match the autoform input type `["cfs-file"]`?